[Window Controls Overlay] Add UI considerations (#7320)
GoogleChrome/web.dev: src/site/content/en/blog/window-controls-overlay/index.md

{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/1VonSc0jOiOureeSkqtf.png", alt="PWA running in standalone mode with the window controls overlay displayed in the body.", width="800", height="428" %}

## UI considerations

> While it may be tempting, it is not recommended to try to create classic dropdown
> menu bars in the Window Controls Overlay area. Doing so would violate the

Reviewer: ```suggestion
While it may be tempting, creating a classic dropdown menu in the Window Controls Overlay area is not recommended. Doing so would violate the
```

Reviewer: This is Strunk and White rule II-22. Place the emphatic words of a sentence at the end. This sounds counterintuitive. It allows the context of a point to sink in before the point lands.

Author: Wow, thanks!

[design guidelines on macOS](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-bar-menus/),

> a platform on which users expect menu bars (that typically include both system-provided menus and

Reviewer: ```suggestion
a platform on which users expect menu bars (that typically includes both system-provided menus and
```

Reviewer: 'Includes' seems to refer to 'platform'.

Author: In this case it refers to menu bars.
